luigi/contrib/azurebatch.py

```diff
@@ -415,42 +441,58 @@ class AzureBatchTask(luigi.Task):
     :param pool_vm_size (str): size of vm to use for the batch job
 
     """
+
     batch_account_name = luigi.Parameter()
     batch_account_key = luigi.Parameter()
+    batch_account_url = luigi.Parameter()
     storage_account_name = luigi.Parameter()
     storage_account_key = luigi.Parameter()
-    input_path = luigi.Parameter()
-    output_path = luigi.Parameter()
+    input_path = luigi.Parameter(default=" ")
+    command = luigi.Parameter(default="echo Hello World")
+    output_path = luigi.Parameter(default=" ")
     pool_id = luigi.Parameter(default=TASK_POOL_ID)
     job_id = luigi.Parameter(default=TASK_JOB_ID)
     pool_node_count = luigi.IntParameter(default=TASK_POOL_NODE_COUNT)
     pool_vm_size = luigi.Parameter(default=TASK_POOL_VM_SIZE)
     kwargs = luigi.DictParameter(default={})
+    container_name = "luigitargetdata"

```

Comment:
This should also be a `luigi.Parameter(default="luigiTargetData")`

Reply:
We are creating this as tmp container & deleting it after, thus though of fixing it. However, yes I will make it a luigi parameter with default

Reply:
Done